mpcstats_lib.py

```diff
@@ -38,12 +38,33 @@ def print_data(data: Matrix):
             print_ln("data[{}][{}]: %s".format(i, j), data[i][j].reveal())
 
 
+def _mean(data: list[sint]) -> (float, int):
+    total = sum(if_else(i != MAGIC_NUMBER, i, 0) for i in data)
+    count = sum(if_else(i != MAGIC_NUMBER, 1, 0) for i in data)
+    return total / count, count
+
+
+def _variance(data: list[sint], use_bessels: bool) -> float:
```

Comment:
We should use sfix for all secret types for consistency
```suggestion
def _variance(data: list[sfix], use_bessels: bool) -> sfix:
```

Reply:
Made the change.